src/soc/ibm/power9/istep_13_8.c

```diff
@@ -596,7 +596,7 @@ static void thermal_throttle_scominit(uint8_t chip, int mcs_i, int mca_i)
 	    // Set to disable permanently due to hardware design bug (HW403028) that won't be changed
 	    [53]    MBA_FARB3Q_CFG_NM_CHANGE_AFTER_SYNC = 0
 	*/
-	printk(BIOS_EMERG, "Please FIXME: ATTR_MSS_RUNTIME_MEM_THROTTLED_N_COMMANDS_PER_SLOT\n");
+	printk(BIOS_DEBUG, "Please FIXME: ATTR_MSS_RUNTIME_MEM_THROTTLED_N_COMMANDS_PER_SLOT\n");
```

Comment:
This was recently changed, you may want to amend the commit and skip it so there will be no conflicts during rebase.